### ISSUE TITLE
bug:  TypeError: Cannot read properties of undefined (reading 'concat')

### ISSUE THREAD

**Issue by lolollllo** [favware/rollup-type-bundler#123]
**Describe the bug**
Undefined term 'concat' in package src

<!-- A clear and concise description of what the bug is. -->

**To Reproduce**

<!-- Steps to reproduce the behavior: -->

1. Execute 'rollup-type-bundler -e stream'
3. See error

**Expected behavior**
Not to throw an error

<!-- A clear and concise description of what you expected to happen. -->
The package gets to be published without the error being thrown

**Screenshots**

<!-- If applicable, add screenshots to help explain your problem. -->
![image](https://user-images.githubusercontent.com/99907738/175758249-1190618c-2c6d-4e4f-ae52-096c527c3b1a.png)

**FULL ERROR**
```file:///home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/dist/cli.js:23
    .option('-e, --external [external...]', `Repeatable, each will be treated as a new entry. Library or libraries to treat as external in Rollup (see: ${cyan('https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency')})`, (value, previous) => previous.concat([value]));
                                                                                                                                                                                                                                                                                  ^
TypeError: Cannot read properties of undefined (reading 'concat')
    at Option.parseArg (file:///home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/dist/cli.js:23:275)
    at handleOptionValue (/home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/node_modules/commander/lib/command.js:539:24)
    at Command.<anonymous> (/home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/node_modules/commander/lib/command.js:566:7)
    at Command.emit (node:events:5[37](https://github.com/lolollllo/discord-portable-player/runs/7051610617?check_suite_focus=true#step:5:38):28)
    at Command.parseOptions (/home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/node_modules/commander/lib/command.js:1[41](https://github.com/lolollllo/discord-portable-player/runs/7051610617?check_suite_focus=true#step:5:42)7:18)
    at Command._parseCommand (/home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/node_modules/commander/lib/command.js:1196:25)
    at Command.parse (/home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/node_modules/commander/lib/command.js:891:10)
    at file:///home/runner/work/discord-portable-player/discord-portable-player/node_modules/@favware/rollup-type-bundler/dist/cli.js:24:25
```

<!-- Add any other context about the problem here. -->

**Comment by favna** [comment]
Ah... That's an oversight caused by a recent change. Crud. I'll get around fixing this tomorrow I think.

**Comment by lolollllo** [comment]
> Ah... That's an oversight caused by a recent change. Crud. I'll get around fixing this tomorrow I think.

Alright.

**Comment by favna** [comment]
Fix released in v1.0.9
